wp-includes/widgets/class-wp-widget-audio.php

```diff
@@ -23,9 +23,10 @@ class WP_Widget_Audio extends WP_Widget_Media {
 	 * @access public
 	 */
 	public function __construct() {
-		parent::__construct( 'audio', __( 'Audio' ), array(
-			'classname'   => 'widget_audio',
+		parent::__construct( 'media-audio', __( 'Audio' ), array(
```

Comment:
We should probably use `_`s :(

https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#naming-conventions